packages/core/components/Puck/styles.module.css

```diff
@@ -194,3 +207,23 @@
   grid-area: right;
   overflow-y: auto;
 }
+
+.Puck-canvas {
+  background: var(--puck-color-grey-11);
+  display: flex;
+  grid-area: editor;
+  flex-direction: column;
+  padding: var(--puck-space-px);
+  overflow: auto;
+}
+
+@media (min-width: 1198px) {
```

Comment:
These `min-width` values can probably change now, as they were calculated as best compromise for the the preview width, given it didn't have it's own viewport context. But not blocking, potential optimisation for later.

Reply:
I think it might be best to do this as a follow on (possibly as part of #269 to avoid bloating this PR)

Reply:
Agree